package.json

```diff
@@ -10,10 +10,11 @@
     "generate:tailwind": "tailwindcss -i ./src/tailwind.css -o ./src/components/Editor/tailwind.generated.css --full --minify",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives",
     "postbuild": "ncp dist/index.html dist/404.html",
-    "prepare": "node -e \"try { require('husky').install() } catch (e) {}\"",
+    "prepare": "husky",
```

Comment:
the `prepare` script that you've replaced has been introduced within the inner source project a while ago and for some good reason (that I haven't had the chance to dig into so far). Are you sure, that we don't need it anymore?

Reply:
1. We used it in `base`, `core` -> `elements` and had a problem because of the install/prepare, but this isn't a library so it won't be imported 
2. There was a new husky version some weeks ago, there are some changes like the one you see here

Reply:
@nmerget it even also seems to have been related to the aspect, of how we've previously referenced those base, core and elements dependencies on localhost enviromments, in which this was a problem. This should be fixed by monorepo already.

original PR within internal repo: /db-ui/db-ui-base/-/merge_requests/51